### PR TITLE
Create nickaldwin.json

### PR DESCRIPTION
**Pull request by nickaldwin**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5936"></a>

### CODE REVIEW

File: nickaldwin.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "nickaldwin",
+  "displayStatsPublic": "true",
```
Comment (Ananya2001-an) [inline nickaldwin.json:3]: Remove this prop. It's not needed anymore.
```suggestion
```

File: nickaldwin.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "type": "personal",
+  "bio": "I'am a fullstack software developer",
+  "avatar": "https://github.com/nickaldwin.png",
```
Comment (Ananya2001-an) [inline nickaldwin.json:6]: same for this.
```suggestion
```

File: nickaldwin.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+   "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/nicklemoncito",
+      "icon": "Fatwitter"
```
Comment (Ananya2001-an) [inline nickaldwin.json:11]: typo in icon name
```suggestion
      "icon": "FaTwitter"
```

File: nickaldwin.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    },
+     {
+      "name": "Follow me on Github",
+      "url": "https://twitter.com/nickaldwin",
```
Comment (Ananya2001-an) [inline nickaldwin.json:15]: you have given url for twitter here instead of github.

**Review by Ananya2001-an — CHANGES_REQUESTED** [review]
Hey @nickaldwin. Thanks for contributing 👍🏼 I have just requested some changes that you have to make. Also rename the file from `nickaldwin.json` to `data/nickaldwin.json`. Your profile should reside inside the data folder.

**Comment by nickaldwin** [comment]
i will change the corrections thank you so much

**Comment by nickaldwin** [comment]
hi @Ananya2001-an i have a question how to update my PR this is my first time

**Comment by Ananya2001-an** [comment]
> hi @Ananya2001-an i have a question how to update my PR this is my first time

You can just push new commits to this branch: `nickaldwin-patch-1` on which the PR is open. That's it :)

**Comment by nickaldwin** [comment]
@Ananya2001-an thank you so much

**Comment by kumarsonsoff3** [comment]
Great collaboration everyone, but I'm closing this PR as it is being a Duplicate of #5938

**Pull request closed without merging**